private/rules/jetifier_maven_map.bzl

```diff
@@ -0,0 +1,126 @@
+jetifier_maven_map = {
+    ("com.android.support", "animated-vector-drawable"): {"28.0.0" : ("androidx.vectordrawable", "vectordrawable-animated", "1.0.0")},
```

Comment:
Should the versions be hardcoded?

Reply:
That's how it is in jetifier. For compatibility reasons it maps one specific version to another since older/new versions aren't guaranteed to be compatible.

Reply:
I see. Thanks.